src/js/components/Tiles.js

```diff
@@ -17,6 +18,9 @@ import CSSClassnames from '../utils/CSSClassnames';
 const CLASS_ROOT = CSSClassnames.TILES;
 const TILE = CSSClassnames.TILE;
 const SELECTED_CLASS = `${TILE}--selected`;
+const MIN_COLUMN_SIZE_LARGE = 384;  // $tile-large-size = 384px
```

Comment:
Is there any way to retrieve this from the DOM after mounting to allow the CSS to drive the sizes? Otherwise this is a bit fragile in case someone changes the CSS variables.